### PR TITLE
Claw code

### PR DESCRIPTION
**Pull request by TheChar**
please check over and add comments and such

### CODE REVIEW

File: src/main/java/frc/robot/Claw.java [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+ */
+public class Claw {
+    //Global Variables
+    public static DoubleSolenoid LPiston, RPiston;
+    public static CANSparkMax claw;
+    public static TalonSRX clawIntakeMotor;
+    
+    //This method should be run first and will initialize all important variables to make the claw work
+    public static void initClaw() {
+        LPiston = new DoubleSolenoid(1, 2); //Creates LPiston | PARAMETERS (forwardChannel, ReverseChannel)
```
Comment (HudsonHurtig) [inline src/main/java/frc/robot/Claw.java]: are these piston ids already used?

**Comment by HudsonHurtig** [comment]
What would happen if you held down both triggers at the same time? there seems to by no way to make sure you don't switch motor direction and output unintentionally. I know we dont have the claw neo on the bot anymore, and we wont flip the robot anymore, but its worth putting in just in case it gets put on again.

**Review by HudsonHurtig — CHANGES_REQUESTED** [review]

File: src/main/java/frc/robot/Claw.java (hunk 1 above)
Comment (TheChar) [reply src/main/java/frc/robot/Claw.java]: I won't know what values to put in the parameters until later. Those are placeholders for now.

Comment (cyrusduong) [reply src/main/java/frc/robot/Claw.java]: Can you check in the cheezy 2019 repo for the correct id's to use here?

File: src/main/java/frc/robot/Claw.java [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+ * 
+ * @author (Charlie Bergdall)
+ * @version (0.0.1) Swenson's Claw uses Neos (Spark MAX), solenoids,
+ */
+public class Claw {
+    //Global Variables
+    public static DoubleSolenoid LPiston, RPiston;
+    public static CANSparkMax claw;
+    public static TalonSRX clawIntakeMotor;
+    private static double scaler = 0.25;
```
Comment (cyrusduong) [inline src/main/java/frc/robot/Claw.java]: Does this ever plan on changing during runtime? (is it going to stay constant)? If so what keyword modifier should we use to indicate this variable as such.

**Review by cyrusduong — DISMISSED** [review]
Nice looking code. Good work very clean and easy to review.

File: src/main/java/frc/robot/Claw.java (hunk 2 above)
Comment (TheChar) [reply src/main/java/frc/robot/Claw.java]: set the value as FINAL. I'll add that now.

File: src/main/java/frc/robot/Claw.java (hunk 1 above)
Comment (TheChar) [reply src/main/java/frc/robot/Claw.java]: Found them as 2 and 5. Should show up in the next commit.

**Review by gtjones02 — DISMISSED** [review]

File: src/main/java/frc/robot/Claw.java (hunk 1 above)
Comment (TheChar) [reply src/main/java/frc/robot/Claw.java]: updated

**Review by gtjones02 — APPROVED** [review]

**Review by Team1710CTO — APPROVED** [review]

**Review by cyrusduong — APPROVED** [review]

**Review by Team1710CTO — APPROVED** [review]

**Review by HudsonHurtig — APPROVED** [review]

**Pull request merged**